core/src/apps/tron/README.md

```diff
@@ -0,0 +1,35 @@
+# TRON
+
+Tron is an EVM like netwrok that allows for nearly free token transfers like `USDT` etc. It achieves this by using `Bandwidth` and `Energy` for transfers, instead of `Gas` in Ethereum. `Bandwidth` is used for native `TRX` transfer, while `Energy` is used for `TRC-20` tokens like `Tether - USDT` etc.
+
+It uses `Protobuff` for encoding its transactions. Each transaction contains its main operation as a `contract`[^1] along with supported data like expiration timestamp, block hash reference[^2], etc.
+
+[`TronWeb` implementation](https://github.com/tronprotocol/tronweb) should be used as source of truth. The documentation on [tron.network](developers.tron.network) and [Tronprotocol](https://github.com/tronprotocol) were incorrect in some places.
+
+## Implementation Notes
+
+Any Transaction begins by sending `TronSignTx` message that contains the header information, including the BIP32 path to the key (e.g. `m/44h/195h/0h/0/0`). Trezor then awaits the contract to be included in the transaction as another message (e.g. `TronTransferContract`). When a valid contract is received, Trezor signs the `raw_data` part of the final transaction and sends the signature in the `TronSignature` message.
+
+## Fees
+
+The `Bandwidth` consumed for processing any transaction is equal to the size of the transaction, i.e. 250 byte transaction = 250 Bandwidth points.
+`Energy` is consumed only on smart contract invocation calls like `TRC-20` tokens are being transferred. If the user has sufficient `Energy` and `Bandwidth` for a transaction, they generally don't have to pay any fee[^3]. If they lack either, some `TRX` is burned to pay for the remaining resource.
+
+### Capping fees
+
+TRON only allows a maximum limit on the `Energy` resource usind the `max_fees` field in any transaction, and not on `TRX` or `Bandwidth`.
+Thus for native TRX transfer via `TransferContract`, we do not show any fees to the user on the device. The host application like Trezor Suite, should display potential resource consumption numbers. We can show the `max_fees` on the device when we start supporting Smart contract calls.
```

Comment:
IIRC, we can actually calculate the Bandwidth needed for a `TransferContract` on the Trezor device side. It seems to be related to the size of the original transaction. We should consider adding a field to display this in the future, just to keep the UI consistent with other chain.